dbt-snowflake/tests/functional/adapter/catalog_integrations/test_catalog_integrations.py

```diff
@@ -36,10 +58,22 @@ def catalogs(self):
     @pytest.fixture(scope="class")
     def models(self):
         return {
-            "models": {
-                "basic_iceberg_table.sql": MODEL__BASIC_ICEBERG_TABLE,
-            }
+            "basic_iceberg_table.sql": MODEL__BASIC_ICEBERG_TABLE,
+            "iceberg_table_with_configs.sql": MODEL__ICEBERG_TABLE_W_CONFIGS,
         }
 
     def test_basic_iceberg_catalog_integration(self, project):
         run_dbt(["run"])
+        iceberg_sql = get_cleaned_model_ddl_from_file("basic_iceberg_table.sql")
```

Comment:
If you're only checking the model file, then I would run `dbt compile` instead of `dbt run`. It should be a bit quicker.

Reply:
Just tried and it looks like 'compile' doesn't result in actually writing out the target files 🤷